### PR TITLE
Modify skipName to not accept numerals as the start of a name

### PR DESCRIPTION
**Pull request by Aflynn50**
This PR modifies skipName to not accept numerals as the start of a name.

### CODE REVIEW

File: internal/parse/parser.go [hunk 1]
```diff
@@ -211,24 +211,33 @@ func (p *Parser) skipString(s string) bool {
 	return false
 }
 
-// isNameByte returns true if the byte passed as parameter is considered to be
-// one that can be part of a name. It returns false otherwise.
+// isNameByte returns true if the byte c can be part of a name. It returns false
+// otherwise.
```
Comment (TheSignPainter98) [inline internal/parse/parser.go]: How about 'the given byte'?

File: internal/parse/parser.go [hunk 2]
```diff
@@ -211,24 +211,33 @@ func (p *Parser) skipString(s string) bool {
...
 func isNameByte(c byte) bool {
 	return 'A' <= c && c <= 'Z' || 'a' <= c && c <= 'z' ||
 		'0' <= c && c <= '9' || c == '_'
 }
 
-// skipName returns false if the parser is not on a name. Otherwise it advances
-// the parser until it is on the first non name byte and returns true.
+// isNameInitialByte returns true if the byte c can appear at the start of a
+// name. It returns false otherwise.
```
Comment (TheSignPainter98) [inline internal/parse/parser.go:214]: Same here perhaps

File: internal/parse/parser.go (hunk 1 above)
Comment (Aflynn50) [reply internal/parse/parser.go]: Changed.

File: internal/parse/parser.go (hunk 2 above)
Comment (Aflynn50) [reply internal/parse/parser.go:214]: Changed.

**Review by niemeyer — APPROVED** [review]
LGTM, thanks.

**Pull request merged**